integration-tests/ledger-signer.spec.ts

```diff
@@ -137,9 +137,9 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
       });
     })
 
-    describe('Should be able to use Ledger with contract API', () => {
+    describe('Test an ability to use Ledger with contract api', () => {
       jest.setTimeout(240000)
-      it('Should originate contract with Ledger', async (done) => {
+      it('As a User I want to verify that I can originate contract with Ledger', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
changed to
'Verify that Ledger returns the correct signature'